src/quo2/components/browser/new_tab/style.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo2.components.browser.new-tab.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def gap
+  {:height 8})
+
+(defn container
+  [customization-color theme]
+  {:width            160
```

Comment:
About width of the new-tab component, it should probably be dynamic to make sure 2 elements fit the browser screen like in the design. But since design is not final I decided to keep it like this for now. Though maybe we can revisit